tv_apps/tv_shared/js/cards/deck.js

```diff
@@ -6,6 +6,10 @@
   var Deck = function Deck(options) {
     this.nativeApp = options.nativeApp;
     this.name = options.name;
+    this.fontIcon = options.fontIcon;
```

Comment:
Since cards are shared by app-deck and smart-home, you should ask dwi2 to sync about this.